mlir/lib/Dialect/ArmSVE/Transforms/LegalizeVectorStorage.cpp

```diff
@@ -27,23 +27,34 @@ using namespace mlir::arm_sve;
 // this pass.
 constexpr StringLiteral kSVELegalizerTag("__arm_sve_legalize_vector_storage__");
 
+/// Definitions:
+///
+/// [1] svbool = vector<...x[16]xi1>, which maps to some multiple of full SVE
```

Comment:
Right, so now I see that `svbool` != `svbool_t` (the latter is defined here: https://arm-software.github.io/acle/main/acle.html#markdown-toc-sve-predicate-types). So, let's use `svbool` consistently in this file. Could you replace `svbool_t` with `svbool`?